Library/Homebrew/test/dev-cmd/generate-cask-ci-matrix_spec.rb

```diff
@@ -8,11 +8,10 @@
 
   it_behaves_like "parseable arguments"
```

Comment:
Does this test actually need the `ENV["GITHUB_REPOSITORY"]` assignment above?

Reply:
Yeah it does, that's why I had to keep it in the original PR

Reply:
Is that because the assertion is in `initialize`? If so, any reason why that check cannot be in `run`?